projects/gloo/cli/pkg/flagutils/install.go

```diff
@@ -15,11 +15,14 @@ func AddInstallFlags(set *pflag.FlagSet, install *options.Install) {
 	set.StringVar(&install.HelmReleaseName, "release-name", constants.GlooReleaseName, "helm release name")
 	set.BoolVar(&install.CreateNamespace, "create-namespace", true, "Create the namespace to install gloo into")
 	set.StringVarP(&install.Namespace, "namespace", "n", defaults.GlooSystem, "namespace to install gloo into")
+	set.StringVar(&install.Release, "gloo-release", "", "Version of Gloo to install (e.g. 1.0.0, defaults to latest)")
```

Comment:
```suggestion
	set.StringVar(&install.Release, "open-source-release", "", "Version of Gloo to install (e.g. 1.0.0, defaults to latest)")
```
idk, this is the kind of thing that might be good for our new design review meetings, to get consensus on a user-facing change like this

Reply:
Adding a --release-version flag.